README.md

```diff
@@ -7,6 +7,20 @@ crowdsourcing workers, and from those judgments, to compute relevance metrics
 such as normalized discounted cumulative gain (NDCG) and mean average precision
 (MAP).
 
+## Running tests
+
+From a fresh arcs virtual environment, install pytest:
+
+```bash
+pip install pytest
```

Comment:
Oh. Not instead, but in addition (since I don't think we should declare test-time dependencies in requirements, although I should see what the Pythonic way to handle this is). Done.